File: app/src/main/java/com/jo/ArduinoInfoSuite/Adapters/LuisLongViewHolder.java
package com.jo.ArduinoInfoSuite.Adapters;

import android.view.View;

class LuisLongViewHolder(view: View):RecyclerView.ViewHolder(view) {
        val superHaciendaName = view.findViewById<TextView>(R.id.tvName)
        val Precio = view.findViewById<TextView>(R.id.tvPrecio)
        val Descripcion = view.findViewById<TextView>(R.id.tvDescripcion)
        val photo = view.findViewById<ImageView>(R.id.ivHacienda)

        fun render(superLuisLongModel: SuperLuisLong){
        superHaciendaName.text = superHaciendaModel.name
        Precio.text = superHaciendaModel.precio
        Descripcion.text = superHaciendaModel.descripcion
        Glide.with(photo.context).load(superHaciendaModel.photo).into(photo)
        }
        }

File: app/src/main/java/com/jo/ArduinoInfoSuite/Adapters/LuisLongViewHolder.kt
package com.jo.ArduinoInfoSuite.Adapters

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.bumptech.glide.Glide
import com.codingstuff.loginandsignup.R
import com.jo.ArduinoInfoSuite.LuisLong.SuperLuisLong

class LuisLongViewHolder(view: View): RecyclerView.ViewHolder(view) {
    val superLuisLongName = view.findViewById<TextView>(R.id.tvNameL)
    val PrecioL = view.findViewById<TextView>(R.id.tvPrecioL)
    val DescripcionL = view.findViewById<TextView>(R.id.tvDescripcionL)
    val photoL = view.findViewById<ImageView>(R.id.ivHaciendaL)

    fun render(superLuisLongModel: SuperLuisLong){
        superLuisLongName.text = superLuisLongModel.nameL
        PrecioL.text = superLuisLongModel.precioL
        DescripcionL.text = superLuisLongModel.descripcionL
        Glide.with(photoL.context).load(superLuisLongModel.photoL).into(photoL)
    }
}